_docs/1.0.0/usage-errcol.md

```diff
@@ -9,11 +9,64 @@ categories:
 Error Column
 ============
 
-`errCol` description will come here
+<!-- toc -->
+- [Description](#description)
+- [Error Types](#error-types)
+  - [stdCastError](#stdcasterror)
+  - [stdNullErr](#stdnullerr)
+  - [confMappingErr](#confmappingerr)
+  - [confCastErr](#confcasterr)
+  - [confNegErr](#confnegerr)
+  - [confLitErr](#confliterr)
+- [Notes](#notes)
+<!-- tocstop -->
 
-### Random notes
+Description
+-----------
 
-Notes not to be forgotten and included when proper documentation is created
+`errCol` is a special, automatically created, composite column, that contains descriptions of all the issues encountered
+during **Standardization** and **Conformance** of the particular row.
 
-* When *Standardization* of a value fails and the column has a `sourcecolumn` *metadata* property defined, the
+It's an array, where each member represents one error that happened on the particular row during its processing.
+The array member is a structured information itself:
+
+* `errType` - string representation of the type of the error
+* `errCode` - code representation of the type of the error in the form of _E#####_, where # is a digit (e.g. E00001)
+* `errMsg` - description of the error
+* `errCol` - name of the column, in which the error happened [\*](#notes-star)
+* `rawValues` - the input values for the error
+* `mappings` - ???
+
+
+Error Types
+-----------
+
+### `stdCastError`
+
+When column value fails to standardize to the expected type
```

Comment:
This list might be a good candidate for a table, e.g.

| Error type | Description |
|:------------------:|:-------------:|
|       stdCastError     |  When column value fails to standardize to the expected type  |
| ... | ... |

Reply:
Yeah, it would definitely be more concise. My original though was there might be more text. But if it would happen, it can be expanded to chapter from table then.